examples/server.rs

```diff
@@ -35,42 +41,55 @@ async fn run_server() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
         Some(ref p) => p.to_owned(),
         None => "1337".to_owned(),
     };
-    let addr = format!("127.0.0.1:{}", port).parse()?;
+    let addr: SocketAddr = format!("127.0.0.1:{}", port).parse()?;
```

Comment:
nit: I think we generally prefer to use a turbofish for this sort of thing.

Reply:
This is a pretty roundabout way of doing it, I'd suggest writing it as `SocketAddr::from(([127, 0, 0, 1], port))`.

Reply:
I'll remove this example since it was previously being used to showcase usage of the `Acceptor`, which I'm removing now.

Reply:
I think it would be nice if you adapted it to the new API you showed in https://github.com/rustls/hyper-rustls/pull/232#issuecomment-1879272874 instead of deleting it, since probably a lot of users will want to know how to do it, especially if they're existing users of the current rustls version and looking for the upgrade path.

Reply:
I've added such an example back and addressed the comments left on this old one there.